src/or/hs_service.c

```diff
@@ -1515,9 +1520,9 @@ build_descriptors_for_new_service(hs_service_t *service, time_t now)
 
   /* Build descriptors. */
   build_service_descriptor(service, now, current_desc_tp,
-                           &service->desc_current);
+                           &service->desc_current, 1);
```

Comment:
If we're using a bool, let's use "true" and "false", not "1" and "0".

Reply:
Done in 286c8e2e34.

Reply:
This is not needed anymore, because of the bugfix in f13d629db0 .